### ISSUE TITLE
404 Error for Simple Network page

### ISSUE THREAD

**Issue by Alumni1506** [iamromulan/quectel-rgmii-toolkit#43]
Hi,

Thank you for your excellent work. Installation of simple admin went smoothly.
The only issue I have is getting a "404 Not Found The requested URL was not found" whenever I try to access the "Simple Network" page.

![image](https://github.com/iamromulan/quectel-rgmii-toolkit/assets/112524737/2c70655d-18dc-4ef7-8fdb-54d69c4c3639)

![image](https://github.com/iamromulan/quectel-rgmii-toolkit/assets/112524737/f1380705-c7be-44a0-90dc-46160f8159e3)

**Comment by dr-dolomite** [comment]
Marked as fixed. 
Reason: Incomplete install.

**Comment by Alumni1506** [comment]
Hi,

I reinstalled simpleadmin 5 times and I'm still getting the same error.
```
Installing Simple Admin...
What version of Simple Admin do you want to install? This will start a webserver on port 8080
1) Stable current version, (Main Branch)
2) Install Test Build (Development Branch)
3) Return to Main Menu
Select your choice:
1
1
Installing simpleadmin from the main stable branch
Removing installed AT Socat Bridge services...
Installing AT Socat Bridge services...
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-armel-static'
socat-armel-static   100% |********************************|  864k  0:00:00 ETA
'socat-armel-static' saved
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'killsmd7bridge'
killsmd7bridge       100% |********************************|   141  0:00:00 ETA
'killsmd7bridge' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'atcmd'
atcmd                100% |********************************|  1775  0:00:00 ETA
'atcmd' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd11.service'
socat-smd11.service  100% |********************************|   364  0:00:00 ETA
'socat-smd11.service' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd11-from-ttyIN.service'
socat-smd11-from-tty 100% |********************************|   301  0:00:00 ETA
'socat-smd11-from-ttyIN.service' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd11-to-ttyIN.service'
socat-smd11-to-ttyIN 100% |********************************|   301  0:00:00 ETA
'socat-smd11-to-ttyIN.service' saved
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-killsmd7bridge.service'
socat-killsmd7bridge 100% |********************************|   164  0:00:00 ETA
'socat-killsmd7bridge.service' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd7-from-ttyIN2.service'
socat-smd7-from-ttyI 100% |********************************|   299  0:00:00 ETA
'socat-smd7-from-ttyIN2.service' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd7-to-ttyIN2.service'
socat-smd7-to-ttyIN2 100% |********************************|   299  0:00:00 ETA
'socat-smd7-to-ttyIN2.service' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'socat-smd7.service'
socat-smd7.service   100% |********************************|   365  0:00:00 ETA
'socat-smd7.service' saved
Adding AT Socat Bridge systemd service units...
AT Socat Bridge service online: smd11 to ttyOUT
AT Socat Bridge service online: smd7 to ttyOUT2
AT Socat Bridge services Installed!
Failed to stop simplefirewall.service: Unit simplefirewall.service not loaded.
Failed to stop ttl-override.service: Unit ttl-override.service not loaded.
Installing/Updating Simple Firewall...
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to '/usrdata/simplefirewall/simplefirewall.sh'
simplefirewall.sh    100% |********************************|   572  0:00:00 ETA
'/usrdata/simplefirewall/simplefirewall.sh' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to '/usrdata/simplefirewall/ttl-override'
ttl-override         100% |********************************|  1509  0:00:00 ETA
'/usrdata/simplefirewall/ttl-override' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to '/usrdata/simplefirewall/ttlvalue'
ttlvalue             100% |********************************|     1  0:00:00 ETA
'/usrdata/simplefirewall/ttlvalue' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to '/usrdata/simplefirewall/systemd/simplefirewall.service'
simplefirewall.servi 100% |********************************|   206  0:00:00 ETA
'/usrdata/simplefirewall/systemd/simplefirewall.service' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to '/usrdata/simplefirewall/systemd/ttl-override.service'
ttl-override.service 100% |********************************|   202  0:00:00 ETA
'/usrdata/simplefirewall/systemd/ttl-override.service' saved
Simple Firewall installation/update complete.
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'simpleadmin_generate_status.service'
simpleadmin_generate 100% |********************************|   191  0:00:00 ETA
'simpleadmin_generate_status.service' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'simpleadmin_httpd.service'
simpleadmin_httpd.se 100% |********************************|   228  0:00:00 ETA
'simpleadmin_httpd.service' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to 'build_modem_status'
build_modem_status   100% |********************************|   961  0:00:00 ETA
'build_modem_status' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'modemstatus_parse.sh'
modemstatus_parse.sh 100% |********************************| 15778  0:00:00 ETA
'modemstatus_parse.sh' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'tojson.sh'
tojson.sh            100% |********************************|  1316  0:00:00 ETA
'tojson.sh' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'atcommander.html'
atcommander.html     100% |********************************| 13087  0:00:00 ETA
'atcommander.html' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to 'index.html'
index.html           100% |********************************| 10532  0:00:00 ETA
'index.html' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to 'speedtest.html'
speedtest.html       100% |********************************|  2451  0:00:00 ETA
'speedtest.html' saved
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'styles.css'
styles.css           100% |********************************|   888  0:00:00 ETA
'styles.css' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'ttl.html'
ttl.html             100% |********************************|  5326  0:00:00 ETA
'ttl.html' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
wget: server returned error: HTTP/1.1 404 Not Found
wget: write error: Broken pipe
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'alpinejs.min.js'
alpinejs.min.js      100% |********************************| 42398  0:00:00 ETA
'alpinejs.min.js' saved
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'admin.css'
admin.css            100% |********************************|  2118  0:00:00 ETA
'admin.css' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'bulma.css'
bulma.css            100% |********************************|  202k  0:00:00 ETA
'bulma.css' saved
Connecting to raw.githubusercontent.com (185.199.111.133:443)
wget: note: TLS certificate validation not implemented
saving to 'get_atcommand'
get_atcommand        100% |********************************|  1059  0:00:00 ETA
'get_atcommand' saved
Connecting to raw.githubusercontent.com (185.199.108.133:443)
wget: note: TLS certificate validation not implemented
saving to 'get_csq'
get_csq              100% |********************************|   210  0:00:00 ETA
'get_csq' saved
Connecting to raw.githubusercontent.com (185.199.110.133:443)
wget: note: TLS certificate validation not implemented
saving to 'get_ttl_status'
get_ttl_status       100% |********************************|   318  0:00:00 ETA
'get_ttl_status' saved
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
saving to 'set_ttl'
set_ttl              100% |********************************|  1349  0:00:00 ETA
'set_ttl' saved
```

**Comment by Alumni1506** [comment]
Hi thanks for getting back.
Any attempt to install the development branch is refused it say it is under development

**Comment by dr-dolomite** [comment]
> Hi thanks for getting back. Any attempt to install the development branch is refused it say it is under development

Go to development branch and use the link from it to download the installer.

**Comment by Alumni1506** [comment]
**Update**
Installing simpleadmin via the development branch works. Thanks

**Comment by iamromulan** [comment]
@Alumni1506 
Apologies, This is now fixed on the main branch https://github.com/iamromulan/quectel-rgmii-toolkit/commit/3962c5cf5a433a1bf66048d3a4343a0eb53d7b09

@dr-dolomite 
 support for Simple Admin v1 is about to end but not 100% until its pushed to main 😄

**Comment by dr-dolomite** [comment]
> @Alumni1506 Apologies, This is now fixed on the main branch [3962c5c](https://github.com/iamromulan/quectel-rgmii-toolkit/commit/3962c5cf5a433a1bf66048d3a4343a0eb53d7b09)
> 
> @dr-dolomite support for Simple Admin v1 is about to end but not 100% until its pushed to main 😄

My bad 😆

**Comment by Alumni1506** [comment]
No problem. When i get the chance, I'll test the main branch installation.
